package.json

```diff
@@ -60,8 +60,8 @@
         "build": "webpack --mode=production --node-env=production",
         "serve": "webpack serve --progress",
         "serve:prod": "http-server dist -p 8080",
-        "format:check": "prettier --config .prettierrc src//**/*.{js,jsx,json,ts} --check",
-        "format": "prettier --config .prettierrc src//**/*.{js,json,ts} --write",
+        "format:check": "prettier --config .prettierrc \"src/**/*.{js,ts}\" --check",
+        "format": "prettier --config .prettierrc \"src/**/*.{js,ts}\"  --write",
```

Comment:
I understand removing `jsx`, but why `json`? There are some json files in the project that still need to be formatted.